pkg/graph/resolvers/plan_discussion.go

```diff
@@ -174,13 +196,30 @@ func CreateDiscussionReply(
 	if err != nil {
 		return nil, err
 	}
+	err = UpdateTaggedHTMLMentionsAndRawContent(ctx, store, &discussionReply.Content, getAccountInformation)
 
-	result, err := store.DiscussionReplyCreate(logger, discussionReply)
-	return result, err
+	if err != nil {
```

Comment:
Same as above. What are your thoughts about tags failing to parse / update stopping the discussion / reply being created?

Reply:
See comment from above! (As well as in `tag.go`)